array/chunk.test.ts

```diff
@@ -0,0 +1,31 @@
+/* eslint-env jest */
+// @ts-ignore ambiguous import
+import chunk from "./chunk.ts";
+
+describe("chunk", () => {
+  it("should return an array of elements split into groups the length of size", () => {
+    expect(chunk(1)(["a", "b", "c", "d"])).toEqual([
+      ["a"],
+      ["b"],
+      ["c"],
+      ["d"]
+    ]);
+    expect(chunk(2)(["a", "b", "c", "d"])).toEqual([
+      ["a", "b"],
+      ["c", "d"]
+    ]);
+    expect(chunk(4)(["a", "b", "c", "d"])).toEqual([["a", "b", "c", "d"]]);
+  });
+
+  it("should return an array of triples when array can't be split evenly, the final chunk will be the remaining elements", () => {
```

Comment:
Consider rephrasing this into:

```suggestion
  it("fills the final chunk with the remaining elements when the given array can't be split evenly", () => {
```